python/svix/api.py

```diff
@@ -227,6 +236,14 @@ class MessageListOptions(ListOptions):
     channel: t.Optional[str] = None
     tag: t.Optional[str] = None
 
+    def to_dict(self) -> t.Dict[str, t.Any]:
+        d = super().to_dict()
+        if self.before is not None:
+            d["before"] = ensure_tz(self.before)
+        if self.after is not None:
+            d["after"] = ensure_tz(self.after)
```

Comment:
The super impl for `to_dict` prunes out None values so we have to guard these calls 😢